testing/py-backcall/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Contributor: Kevin Daudt <kdaudt@alpinelinux.org>
+# Maintainer: Kevin Daudt <kdaudt@alpinelinux.org>
+pkgname=py-backcall
+pkgver=0.1.0
+pkgrel=0
+pkgdesc="Backwards compatible callback APIs"
+url="https://github.com/takluyver/backcall"
+arch="noarch"
+license="BSD-3-Clause"
+makedepends="python2-dev python3-dev
+options="!check" # pytest broken on alpine
```

Comment:
what is broken? does it needs fix?

Reply:
https://bugs.alpinelinux.org/issues/9420

Reply:
Then we need it for both versions https://github.com/alpinelinux/aports/pull/5095